stable/hoard/templates/deployment.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "hoard.fullname" . }}
+  labels:
+    app: {{ template "hoard.name" . }}
```

Comment:
also please use the new labels, see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata

Reply:
done